rules/linux/command_and_control_cat_network_activity.toml

```diff
@@ -137,11 +137,18 @@ tags = [
         ]
 type = "eql"
 query = '''
-sequence by host.id, process.entity_id with maxspan=1s
-  [process where host.os.type == "linux" and event.action == "exec" and event.type == "start" and process.name == "cat" and 
+sequence by process.entity_id with maxspan=1s
+  [process where event.action == "exec" and event.type == "start" and process.name == "cat" and 
```

Comment:
Nit: any specific observations to remove the host.os.type == "linux" from the query sequence.

Reply:
I copy pasted the endpoint rule variant to detection rules, as that one had additional tunings built-in. Another good catch, thx!